.gitmodules

```diff
@@ -129,3 +129,6 @@
 [submodule "tools/wgsim/src"]
 	path = tools/wgsim/src
 	url = https://github.com/lh3/wgsim.git
+[submodule "tools/cawlign/src"]
```

Comment:
Nitpick: could you move this up to be before coreutils?

Reply:
Done in: https://github.com/niemasd/biowasm/commit/49d42816adffa7ceefed53fc4d82f2eb8b5cbada